drake/solvers/MathematicalProgram.h

```diff
@@ -46,15 +46,6 @@ class DRAKEOPTIMIZATION_EXPORT MathematicalProgramSolverInterface {
   virtual bool available() const = 0;
   virtual bool solve(OptimizationProblem& prog) const = 0;
 };
-
-class DRAKEOPTIMIZATION_EXPORT MathematicalProgramSNOPTSolver :
```

Comment:
Is the longer-term goal to get rid of MathematicalProgramSolverInterface altogether?

(asking for context, not requesting any change)